docs/install/operator/configuring-eventing-cr.md

```diff
@@ -182,5 +180,5 @@ metadata:
   name: knative-eventing
   namespace: knative-eventing
 spec:
-  defaultBrokerClass: MTChannelBasedBroker
+  default-broker-class: MTChannelBasedBroker
```

Comment:
https://github.com/knative/eventing-operator/blob/release-0.14/config/300-eventing-v1alpha1-knativeeventing-crd.yaml#L75

isn't that `defaultBrokerClass` ?

Reply:
Yeah we need to change the field name in the CRD file and in the Go code as well: https://github.com/knative/eventing-operator/blob/release-0.14/pkg/apis/eventing/v1alpha1/knativeeventing_types.go#L75

Reply:
Ok saw https://github.com/knative/eventing-operator/pull/200 now

Reply:
Let's hold it for now, since we do not change it for 14.